src/actions/widgets/index.js

```diff
@@ -0,0 +1,5 @@
+export const CLOCK_TICK = 'clockTick'
+
+const clockTick = time => ({ type: CLOCK_TICK, time })
+
+export { clockTick }
```

Comment:
There will be lots of actions being exported here later, so we are setting up the scaffolding for the things to come